dracut.sh

```diff
@@ -1736,6 +1736,11 @@ if ! [[ -d $dracutsysrootdir$tmpfilesdir ]]; then
     [[ -d $dracutsysrootdir/usr/lib/tmpfiles.d ]] && tmpfilesdir=/usr/lib/tmpfiles.d
 fi
 
+[[ -d $dracutsysrootdir$libexecdir ]] \
+    || libexecdir=$(pkg-config libexecdir --variable=libexecdir 2> /dev/null)
+
+[[ -d $dracutsysrootdir$libexecdir ]] || libexecdir=/usr/libexec
```

Comment:
There is no such thing as a `libexecdir` pkg-config...

```suggestion
if ! [[ -d $dracutsysrootdir$libexecdir ]]; then
    if [[ -d $dracutsysrootdir/usr/libexec ]]; then
        libexecdir=/usr/libexec
    else
        libexecdir=/usr/lib
    fi
fi    
```